main/EDParamsMod.F90

```diff
@@ -54,11 +64,21 @@ module EDParamsMod
    real(r8),protected,public  :: q10_mr     ! Q10 for respiration rate (for soil fragmenation and plant respiration)    (unitless)
    real(r8),protected,public  :: q10_froz   ! Q10 for frozen-soil respiration rates (for soil fragmentation)            (unitless)
 
-   ! two special parameters whose size is defined in the parameter file
+   ! parameters whose size is defined in the parameter file
    real(r8),protected,allocatable,public :: ED_val_history_sizeclass_bin_edges(:)
    real(r8),protected,allocatable,public :: ED_val_history_ageclass_bin_edges(:)
    real(r8),protected,allocatable,public :: ED_val_history_height_bin_edges(:)
    real(r8),protected,allocatable,public :: ED_val_history_coageclass_bin_edges(:)
+
+   ! Switch that defines the current pressure-volume and pressure-conductivity model
+   ! to be used at each node (compartment/organ)
+   ! 1  = Christofferson et al. 2016 (TFS),   2 = Van Genuchten 1980
+   integer, protected,allocatable,public :: hydr_wtftype_node(:) 
+   
+   character(len=param_string_length),parameter,public :: ED_name_vai_top_bin_width = "fates_vai_top_bin_width"
+   character(len=param_string_length),parameter,public :: ED_name_vai_width_increase_factor = "fates_vai_width_increase_factor"
+   character(len=param_string_length),parameter,public :: ED_name_photo_temp_acclim_timescale = "fates_photo_temp_acclim_timescale"
+   character(len=param_string_length),parameter,public :: ED_name_hydr_wtftype_node = "fates_hydr_wtftype_node"
```

Comment:
But why is it called 'wtf' type?

Reply:
water transfer function types, a generalized way of saying pressure volume and pressure conductance relationships. It has no precedent.

Reply:
Probably best to not use WTF in our culture. Unless you want to be teased a bit. I know of an organization that changed their acronym away from it.

Reply:
ok, yeah, I see it now.  I will change to HTF.